doc/release/0.5.0-notes.rst

```diff
@@ -3,13 +3,100 @@ PyWavelets 0.5.0 Release Notes
 ==============================
 
 .. note:: PyWavelets 0.5.0 is not released yet!
+.. |MATLAB|  unicode:: MATLAB U+02122 .. with trademark sign
 
 .. contents::
 
+PyWavelets is a Python toolbox implementing both discrete and continuous
+wavelet transforms with a wide range of built-in wavelets.  C/Cython are used
+for the low-level routines, enabling high performance.  Results of the discrete
+and continuous transforms and the behaviour of the edge modes have been
+validated by comparison to results from |MATLAB|'s Wavelet Toolbox.
```

Comment:
Hmm, not quite sure about this last sentence. I would mention instead a short list of main features, one of which is that when multiple valid implementation choices are possible, we choose for consistency with the Matlab implementations.